src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.hpp

```diff
@@ -562,4 +562,36 @@ class jit_subtract_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const;
 };
 
+class jit_erf_emitter : public jit_emitter {
```

Comment:
@mahdi-jfri Hello! Thank you for the changes!

Minor:
May I ask you to move your `Erf`-related changes to the appropriate places in order to align them with the alphabetical order (in the files jit_eltwise_emitters.hpp, jit_eltwise_emitters.cpp, eltwise.cpp and jit_uni_eltwise_generic.cpp? 😊 It would be great if all implementations are aligned